File: Selenium/Project/Activity4.java
package SelJava;

import io.github.bonigarcia.wdm.WebDriverManager;
import org.openqa.selenium.By;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.firefox.FirefoxDriver;
import org.testng.Assert;
import org.testng.annotations.AfterClass;
import org.testng.annotations.BeforeClass;
import org.testng.annotations.Test;

public class Activity4
{
    WebDriver driver;

    @BeforeClass
    public void beforeMethod()
    {
        WebDriverManager.firefoxdriver().setup();
        driver = new FirefoxDriver();

        driver.get("https://alchemy.hguy.co/jobs");
    }

    @Test
    public void exampleTestCase()
    {
        String abc=driver.findElement(By.tagName("h2")).getText();
        System.out.println("second heading of webpage : " + abc);
        Assert.assertEquals(abc, "Quia quis non");
    }

    @AfterClass
    public void afterMethod()
    {
        driver.quit();
    }
}

File: Selenium/Project/Activity4.py
from selenium import webdriver
from selenium.webdriver.common.by import By
from selenium.webdriver.firefox.service import Service as FirefoxService
from webdriver_manager.core import driver
from webdriver_manager.firefox import GeckoDriverManager

service = FirefoxService(GeckoDriverManager().install())
with webdriver.Firefox(service=service) as driver:
    expected="Quia quis non"
    driver.get("https://alchemy.hguy.co/jobs")
    abc=driver.find_element(By.TAG_NAME,"h2").text
    print("second heading of webpage :" + abc)
    if expected == abc:
        print("webpage 2nd heading and expected  heading  are same")
        driver.quit()
    else:
        print("webpage 2nd heading and expected  heading  are not same")